R/html_dependency.R

```diff
@@ -3,12 +3,8 @@ html_dependency_mousetrap <- function() {
   htmltools::htmlDependency(
     name = "mousetrap",
     version = "1.6.5",
-    package = "keys",
-    src = c(
-      file = "mousetrap",
-      href = "https://unpkg.com/mousetrap@1.6.5"
-    ),
-    script = "mousetrap.min.js"
+    src = system.file("mousetrap", package = "keys"),
+    script = c("mousetrap.min.js", "moustrap-pause.min.js")
```

Comment:
I wanted to get your thoughts on the approach you suggest here compared to [this approach](https://github.com/gadenbuie/applause#create-html_dependency_applause). I thought there might be benefit to showing the src of the javascript both local and on CDN but not actually sure if there is.

Reply:
After addressing the typo in file `moustrap-pause.min.js`, please update the typo here as well.

Reply:
Hey, 

You can choose to do both, as in https://github.com/ColinFay/glouton/blob/master/R/cookies.R#L9

The issue with relying on a CDN is that if the shiny app doesn't have access to the internet, it won't be able to load the script, whereas bundling the script inside the package makes it more reliable. Given that `{shiny}` apps might be used in contexts where the computer doesn't have internet, it might be a good solution. 

Also, CDN might (but that's very rare) be subject to external attackers that can modify the script, creating a security vulnerability in your app. 

On the other hand, using a CDN makes things faster as it's cached by the browser... so... I don't know what's best :) But there should definitely be a backup plan in case the computer doesn't have access to the web.